docs/options.rst

```diff
@@ -41,6 +41,8 @@ General Options:
                                   Defaults to ``'user_claims'``.
 ``JWT_CLAIMS_IN_REFRESH_TOKEN``   If user claims should be included in refresh tokens.
                                   Defaults to ``False``.
+``JWT_ERROR_MESSAGE_KEY``         The key of the error message in a JSON error response.
```

Comment:
Could we add some text in here along the lines of `when using the default error handlers`?